src/main/java/Serf.java

```diff
@@ -0,0 +1,100 @@
+import java.util.Scanner;
+public class Serf {
+    public static void main(String[] args) {
+        String VERTICALLINES = "    ____________________________________________________________";
+        String FIVEWHITESPACES = "     ";
+        Task[] taskList = new Task[100];
+        int taskCounter = 0; // tracks number of tasks in taskList
+        String chatBotName = "Serf";
+        String GREETINGMESSAGE = VERTICALLINES
+                + "\n"
+                + FIVEWHITESPACES
+                + "Good day my lord! I'm " + chatBotName + "\n"
+                + FIVEWHITESPACES
+                + "How shall I serve thee?\n"
+                + VERTICALLINES;
+        String FAREWELLMESSAGE = VERTICALLINES
+                + "\n"
+                + FIVEWHITESPACES
+                + "Farewell, my lord. \n"
+                + VERTICALLINES;
+        System.out.println(GREETINGMESSAGE);
+        Scanner requestedMessage = new Scanner(System.in);
+        String receivedMessage = requestedMessage.nextLine();
+
+        while(!receivedMessage.equals("bye")) {
+            if (receivedMessage.equals("list") && taskCounter > 0) { // list is not empty
```

Comment:
Could be better to replace if-else statements with switch cases.

Reply:
Thank you for your suggestion. I have made changes to my code based on your suggestion in my latest commit tagged "A-CodeQuality".